src/es/components/molecules/figure/Figure.js

```diff
@@ -79,30 +79,34 @@ export default class KsFigure extends Shadow(HTMLElement) {
 
       :host figure {
         margin: 0;
+        width: 100%;
       }
 
       /* line before caption */
       :host figcaption::before {
         content: "";
-        display: block;
+        display: ${this.getAttribute("with-line") !== null ? "block" : "none"};
```

Comment:
Gäbe auch die Methods hasAttribute, das wäre klarer denk ich

Reply:
Habs geupdated :)